earth_enterprise/src/third_party/mod_wsgi/conf.d/mod_wsgi-ge.conf.in

```diff
@@ -7,7 +7,7 @@ LoadModule wsgi_module modules/mod_wsgi.so
     WSGIPythonHome /usr
 
     # Handles Fusion requests.
-    WSGIDaemonProcess ge_fusion_serve threads=15 display-name=%{GROUP} python-path=/opt/google/gepython/Python-2.7.6/lib/python2.7/site-packages:/opt/google/gehttpd/wsgi-bin home=/tmp
+    WSGIDaemonProcess ge_fusion_serve threads=15 display-name=%{GROUP} python-path=/opt/google/gepython/Python-@PYTHON_VERSION@/lib/python2.7/site-packages:/opt/google/gehttpd/wsgi-bin home=/tmp
```

Comment:
We could also replace '2.7' (use mod_wsgi_env['python_version']) in .. lib/python2.7.